src/business.ts

```diff
@@ -114,4 +129,54 @@ export class BusinessApi {
     const success = !resp?.payload?.errorCode
     return { success, body: { ...resp.payload } }
   }
+
+  /*
+   * Function to take the attributes of a new Business account, create that
+   * in Marqeta, and return the account information. If no attributes are
+   * provided, a new Business account will still be created and its
+   * token ID returned.
+   */
+  async create(create: {
+     business: Business
+  }): Promise<{
+    success: boolean,
+    body?: Business,
+    error?: MarqetaError,
+  }> {
+    const searchOptions = snakecaseKeys(create?.business)
+    const resp = await this.client.fire(
+      'POST',
+      'businesses',
+      {},
+      { ...searchOptions }
+    )
+    // catch 404s...
+    if (resp?.payload?.errorCode >='404000' && resp?.payload?.errorCode <= 404999) {
```

Comment:
A few things on this line... first, the spacing on the first inequality:

```typescript
    if (resp?.payload?.errorCode >= '404000' && resp?.payload?.errorCode <= 404999) {
```

there should be a space after the inequality.

Second, the first inequality is testing against a string, and the second against an integer - which is it? `string` or `number`? If it's a `number`, and I suspect it is from the docs, then why not:

```typescript
    if (resp?.payload?.errorCode/1000 == 404) {
```

I mean it's possible to test both ends of the range... but it might be easier to read.

But there's more... 

Why are you catching the 404s? Is there something about these, as opposed to anything 400 or bigger? I mean why not just have:

```typescript
     if (resp?.payload?.errorCode >= 400000){
       const error = resp?.payload?.errorMessage
       return {
         success: false,
         error: {
           type: 'marqeta',
           error,
           status: resp?.payload?.errorCode,
           marqetaStatus: resp?.payload?.errorCode
         },
       }
     }
```

and not have anything special about the 404s... because you're not handling them differently.

If it's because Peach handled them differently, that's because to Peach, getting a list of nothing was a 404... and I didn't want that to be an error. It was perfectly normal to ask for the Businesses, for instance, and get back the empty array.

Peach wanted to say it was a 404. I disagreed.

So why are you catching the 404s?

Reply:
Nothing is different with Marqeta 404s.  I'll adjust for anything > 400000.